python/src/IndependentMetropolisHastings_doc.i.in

```diff
@@ -0,0 +1,77 @@
+%feature("docstring") OT::IndependentMetropolisHastings
+"Random Walk Metropolis-Hastings method.
+
+Refer to :ref:`bayesian_calibration`, :ref:`metropolis_hastings`.
```

Comment:
you forgot to add it to the toctree of some rst file (in python/doc)

Reply:
Done!